ext/RMagick/rmfill.c

```diff
@@ -276,11 +276,11 @@ horizontal_fill(
         master[y].opacity = OpaqueOpacity;
     }
 
-    for (x = 0; x < image->columns; x++)
+    for (x = 0; x < (ssize_t) image->columns; x++)
```

Comment:
unnecessary type cast.
```suggestion
    for (x = 0; x < image->columns; x++)
```